sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -2835,6 +2835,82 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession {
     checkAnswer(df, Row(1, 3, 4) :: Row(2, 3, 4) :: Row(3, 3, 4) :: Nil)
   }
 
+  test("SPARK-27986: support filter clause for aggregate function with hash") {
```

Comment:
we don't need the prefix now: https://github.com/apache/spark-website/pull/231

Reply:
OK. I will remove the prefix in new PR.